rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

```diff
@@ -2,6 +2,31 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[november-3-2023]]
+== November 3, 2023
+
```

Comment:
Is the "What's New" section missing? As you have fixed issues, you may say in such a section something like "This new version fixes issues and updates dependencies", or something similar.